<?xml version="1.0" encoding="UTF-8"?>
<robot generator="Robot 3.2.2 (Python 3.6.8 on win32)" generated="20201220 17:36:05.812" rpa="false">
<suite id="s1" name="Rb" source="D:\code\robot\test\rb.robot">
<test id="s1-t1" name="Valid Login">
<kw name="Open Browser To Login Page">
<msg timestamp="20201220 17:36:05.901" level="FAIL">No keyword with name 'Open Browser To Login Page' found.</msg>
<status status="FAIL" starttime="20201220 17:36:05.901" endtime="20201220 17:36:05.901"></status>
</kw>
<kw name="Close Browser" type="teardown">
<msg timestamp="20201220 17:36:05.908" level="FAIL">No keyword with name 'Close Browser' found.</msg>
<status status="FAIL" starttime="20201220 17:36:05.908" endtime="20201220 17:36:05.909">No keyword with name 'Close Browser' found.</status>
</kw>
<status status="FAIL" starttime="20201220 17:36:05.896" endtime="20201220 17:36:05.919" critical="yes">No keyword with name 'Open Browser To Login Page' found.

Also teardown failed:
No keyword with name 'Close Browser' found.</status>
</test>
<doc>A test suite with a single test for valid login.

This test has a workflow that is created using keywords in
the imported resource file.</doc>
<status status="FAIL" starttime="20201220 17:36:05.827" endtime="20201220 17:36:05.933"></status>
</suite>
<statistics>
<total>
<stat pass="0" fail="1">Critical Tests</stat>
<stat pass="0" fail="1">All Tests</stat>
</total>
<tag>
</tag>
<suite>
<stat pass="0" fail="1" id="s1" name="Rb">Rb</stat>
</suite>
</statistics>
<errors>
<msg timestamp="20201220 17:36:05.888" level="ERROR">Error in file 'D:\code\robot\test\rb.robot' on line 14: Resource file 'resource.robot' does not exist.</msg>
</errors>
</robot>
